libdevcore/CommonData.cpp

```diff
@@ -35,13 +35,26 @@
 using namespace std;
 using namespace dev;
 
-bool dev::isHex(string const& _s)
+namespace
 {
-	unsigned i = (_s.size() >= 2 && _s.substr(0, 2) == "0x") ? 2 : 0;
-	for (; i < _s.size(); ++i)
-		if (fromHex(_s[i], WhenError::DontThrow) == -1)
-			return false;
-	return true;
+int _fromHex(char _i) noexcept
```

Comment:
Let's change the name.    The underscore will have everybody who reads the code asking what the underscore means.

Better to just have a more explicit, longer name, which describes what exactly the function is.